### DISCUSSION TITLE
Uploading translations from one repository to another

### DISCUSSION THREAD

**Discussion by Misat11** [WeblateOrg/weblate#5717] [Q&A] [answered]
Hello,

I've setup my own weblate instance and I want to read base language file from the one repository and then push language files to another repository. In the second repository I want only the language files but the weblate pushes everything including the source code from the first repository. Is there something I missed and I have to change or do I have to modify something in the code?

Thanks for any help,
Misat11

**Comment by nijel** [comment, marked as answer]
Weblate does not do any other changes to the repository, so it pushes it as it is.

You can use git submodules to split the repo, see https://docs.weblate.org/en/latest/faq.html#how-can-i-limit-weblate-access-to-only-translations-without-exposing-source-code-to-it

**Reply by Misat11** [reply]
Got it thanks
